Readme.md

```diff
@@ -790,6 +790,23 @@ try {
 }
 ```
 
+By default Commander is configured for a command-line application and writes to stdout and stderr.
+You can modify this behaviour for custom applications. In addition, you can modify the display of error messages.
+
+Example file: [configure-output.js](./examples/configure-output.js)
+
+
+```js
+program
+  .configureOutput({
+    // Visibly override write routines as example!
+    writeOut: (str) => process.stdout.write(`[OUT] ${str}`),
+    writeErr: (str) => process.stdout.write(`[ERR] ${str}`),
+    // Output errors in red.
+    outputError: (str, write) => write(red(str))
```

Comment:
I found it difficult to understand the `red` function because it came out of nowhere.

How about including the following in the readme code examples as well?

```js
function red(str) {
  return `\x1b[31m${str}\x1b[0m`;
}
```

Reply:
Good feedback, will do. 👍

Reply:
Switched name to `errorColor` (which was what I called this in one of my own programs!) and added implementation to README as suggested.